langs/elm/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM alpine:3.16 as builder
+
+RUN wget -O - 'https://github.com/elm/compiler/releases/download/0.19.1/binary-for-linux-64-bit.gz' \
+    | gunzip -c >/usr/local/bin/elm
+RUN chmod +x /usr/local/bin/elm
+RUN apk add --no-cache nodejs=16.17.1-r0
+
+
```

Comment:
Usually on the CG docker images, in the `builder`, you install whatever needs installed, then do a second stage - `FROM codegolf/lang-base`, and then copy over whichever files and libraries you need to with `COPY --from=0`. Is this possible with Elm?

Reply:
I haven't succeeded to build the image like that. It annoys me too because I don't see a reason why it shouldn't be possible.

Reply:
I'll try later. You usually have to copy a bunch of stuff from /lib and /usr/lib.